install.sh

```diff
@@ -193,7 +177,7 @@ venv_setup ()
     else
         reinstall='true'
     fi
-    virtualenv -p "${python}" "${jm_source}/jmvenv" || return 1
+    "${python}" -m venv "${jm_source}/jmvenv"
```

Comment:
Shouldn't we check success here the same way as with virtualenv?
```suggestion
    "${python}" -m venv "${jm_source}/jmvenv" || return 1
```

Reply:
Fixed in 0ff6148a6e4b265e7cc36f922085fec2c2031528